### ISSUE TITLE
Bump COA CF CLI version to also include CF7 and refine broker smoke tests incl cmdb

### ISSUE THREAD

**Issue by gberche-orange** [orange-cloudfoundry/paas-templates#773]
### Expected behavior

* As a paas-templates author 
* In order to better test cmdb features, including dynamic catalog
* I need the CF CLI V7 features that better support multi service registration, see 
   * https://github.com/cloudfoundry/cli/issues/1849#issuecomment-618440031
      * [**v7/v8** cf marketplace -s: Show details of service plans for a service offering when user not logged in](https://www.pivotaltracker.com/story/show/172127129)
      * [**v7/v8** cf marketplace: Show only service offerings list (--no-plans flag)](https://www.pivotaltracker.com/story/show/172127112)

### Observed behavior
 
[COA image version](https://github.com/orange-cloudfoundry/cf-ops-automation/blob/71b2837fbe7d6e05359929f96f98ecfc872ef40f/concourse/tasks/post_bosh_deploy.yml#L2-L7)

```bash
#fly into a container
# fly -t fe-int-ops i -u https://elpaaso-concourse.redacted-domain.org/teams/ops-depls/pipelines/ops-depls-cf-apps-generated/jobs/cf-push-osb-cmdb-broker/builds/7

bash-4.4# cf -v
cf version 6.47.2+d526c2cb3.2019-11-05
bash-4.4# cf7
bash: cf7: command not found
bash-4.4# which cf
/usr/local/bin/cf
bash-4.4# ls -al /usr/local/bin/c
cf       credhub  
bash-4.4# ls -al /usr/local/bin/c*
-rwxr-xr-x    1 root     root      21916448 Nov  5 01:44 /usr/local/bin/cf
-rwxr-xr-x    1 root     root      11610054 Oct  4  2019 /usr/local/bin/credhub

```

### Steps

* [x] Upgrade cf app manifests in paas-templates to be compatible to V7 https://github.com/orange-cloudfoundry/paas-templates-private/pull/218
* [x] Update cf deployment > 13 to be compliant with cf cli v7 https://github.com/orange-cloudfoundry/paas-templates/issues/842
* [x] Bump CF CLI version in COA https://github.com/orange-cloudfoundry/cf-ops-automation/issues/329 
* [ ] Validate that current paas-templates parsings of CF CLI output are not broken: in particular common-broker-scripts

### Affected releases

* 46.x

**Comment by o-orand** [comment]
Included in [COA 5.1 ](https://github.com/orange-cloudfoundry/cf-ops-automation/milestone/19?closed=1)

Provided- by /orange-cloudfoundry/cf-ops-automation](#337)

**Comment by o-orand** [comment]
invalid CLI commands are details here: #1047
